website/screens/components/radio-group/code/RadioGroupCodePage.tsx

```diff
@@ -0,0 +1,227 @@
+          <td>
+            <Code>false</Code>
+          </td>
+          <td>
+            If true, the radio group will be optional, showing{" "}
+            <Code>(Optional)</Code> next to the label and adding a default last
+            option with an empty string as value. Otherwise, the field will be
+            considered required and an error will be passed as a parameter to
+            the OnBlur functions if an option wasn't selected.
+          </td>
+        </tr>
+        <tr>
+          <td>optionalItemLabel: string</td>
+          <td>
+            <Code>"N/A"</Code>
+          </td>
+          <td>Label of the optional radio input.</td>
+        </tr>
+        <tr>
+          <td>readonly: boolean</td>
+          <td>
+            <Code>false</Code>
+          </td>
+          <td>If true, the component will be marked as readonly.</td>
+        </tr>
+        <tr>
+          <td>stacking: "row" | "column"</td>
+          <td>
+            <Code>"column"</Code>
+          </td>
+          <td>Sets the orientation of the options within the radio group.</td>
+        </tr>
+        <tr>
+          <td>onChange: function</td>
+          <td></td>
+          <td>
+            This function will be called when the user chooses an option. The
+            new value will be passed to this function.
+          </td>
+        </tr>
+        <tr>
+          <td>onBlur: function</td>
+          <td></td>
+          <td>
+            This function will be called when the radio group loses the focus.
+            An object including the value and the error will be passed to this
+            function. An example of this object is: {"{ "}
+            <Code>value: value, error: error</Code>
+            {" }"}. If there is no error, error will not be defined.
+          </td>
+        </tr>
+        <tr>
+          <td>error: string</td>
+          <td></td>
+          <td>
+            If it is a defined value and also a truthy string, the component
+            will change its appearance, showing the error below the radio group.
+            If the defined value is an empty string, it will reserve a space
+            below the component for a future error, but it would not change its
+            look. In case of being undefined or null, both the appearance and
+            the space for the error message would not be modified.
+          </td>
+        </tr>
+        <tr>
+          <td>tabIndex: number</td>
+          <td>
+            <Code>0</Code>
+          </td>
+          <td>Value of the tabindex attribute.</td>
+        </tr>
+        <tr>
+          <td>ref: object</td>
+          <td></td>
+          <td>Reference to the component.</td>
+        </tr>
+      </DxcTable>
+    ),
+  },
+  {
+    title: "Examples",
+    subSections: [
+      {
+        title: "Basic usage",
+        content: (
+          <>
+            <Example example={basicUsage} defaultIsVisible />
+          </>
+        ),
+      },
+      {
```

Comment:
Basic usage example is not needed here, as basic usage should be controlled and it is the same as the controlled example here.

Reply:
Done